### PR TITLE
[Term Entry] Python:Pandas dataframe at #4520

### LINKED ISSUE #4520: [Term Entry] Python:Pandas dataframe at
### Reason/inspiration (optional)

"We would like a new term entry in the `dataframe` concept for [Pandas](https://www.codecademy.com/resources/docs/pandas): at. The entry should go in a new file under `docs/content/pandas/concepts/dataframe/terms/at/at.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use the trace function to find the trace of a square matrix.
- Should contain information for functions and syntax.
- Should have a Python codebyte with a working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by aSagCoder**
### Description

- Created a new branch in order to make merging easier for the given term entry. Has already undergone two reviews and just needs to get merged into the main branch. 
- Added the .at[] function for Pandas.
- Can be located here - docs/content/pandas/concepts/dataframe/terms/at/at.md.

### Issue Solved
Closes #4520 

### Type of Change

- Adding a new entry
- ~~Editing an existing entry (fixing a typo, bug, issues, etc)~~
- ~~Updating the documentation~~

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pandas/concepts/dataframe/terms/at/at.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - 'Data Science'
+Tags: 
+  - 'Data Structures'
+  - 'Pandas'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In Pandas, the **`.at[]`** function is used as an accessor to fetch a specific value from a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) using row/column pairs.
```
Comment (SaviDahegaonkar) [inline content/pandas/concepts/dataframe/terms/at/at.md:15]:
``` suggestion
In Pandas, the **`.at[]`** function is used as an accessor to fetch a specific value from a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) using row and column pairs.
```
Made a slight change for `row/column pairs`. In pandas for the `.at[ ]` function fetch using both row and column pairs, the use of the `/` symbol shows that the `.at[ ]` function either uses rows or columns to fetch a specific value.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @aSagCoder,
I had a look at your file and have suggested a few changes. Please do it as soon as possible.

Thanks,
Savi

**Comment by aSagCoder** [comment]
@SaviDahegaonkar Hello there! I made the necessary changes that you suggested. If there's anything else you want me to do, just ping me!

**Comment by mamtawardhani** [comment]
Hey @SaviDahegaonkar please check this on priority, since this issue is very old.

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @aSagCoder ,
I reviewed your file and you did all the necessary changes. I hereby approve it.

**Comment by SaviDahegaonkar** [comment]
> Hey @SaviDahegaonkar please check this on priority, since this issue is very old.

Should I merge this PR?

**Review by mamtawardhani — APPROVED** [review]
Thank you for your contribution @aSagCoder 😄 

The entry is ready to be merged! 🚀

**Pull request merged**